CMakeLists.txt

```diff
@@ -243,8 +243,9 @@ endfunction()
 if (CMAKE_CONFIGURATION_TYPES) # multi-configuration generator?
     set(CMAKE_CONFIGURATION_TYPES "Debug;Checked;Release;RelWithDebInfo" CACHE STRING "" FORCE)
 endif (CMAKE_CONFIGURATION_TYPES)
-set(CMAKE_C_FLAGS_CHECKED "")
-set(CMAKE_CXX_FLAGS_CHECKED "")
+
+set(CMAKE_C_FLAGS_CHECKED ${CLR_C_FLAGS_CHECKED_INIT} CACHE STRING "Flags used by the compiler during checked builds.")
+set(CMAKE_CXX_FLAGS_CHECKED ${CLR_CXX_FLAGS_CHECKED_INIT} CACHE STRING "Flags used by the compiler during checked builds.")
 set(CMAKE_EXE_LINKER_FLAGS_CHECKED "")
```

Comment:
I have originally considered that, since it is not necessary - these variables are not defined at all and so if anyone used those, they would behave as if they were empty. But I have decided to keep them in just as a place where if someone needed to modify them, there was already a place ready for that.
Does it make sense?